e2e/cypress/integration/integrations/incoming_webhook/slack_formatting_spec.js

```diff
@@ -0,0 +1,325 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @incoming_webhook
+
+describe('Incoming webhook', () => {
+    let testTeam;
+    let testChannel;
+    let incomingWebhook;
+    let offTopicLink;
+    let sysadminUser;
+
+    before(() => {
+        cy.apiUpdateConfig({
+            ServiceSettings: {
+                EnablePostUsernameOverride: true,
+                EnablePostIconOverride: true,
+            },
+        });
+
+        // # Create and visit new channel and create incoming webhook
+        cy.apiInitSetup().then(({team, channel, user}) => {
+            testTeam = team;
+            testChannel = channel;
+
+            const newIncomingHook = {
+                channel_id: channel.id,
+                channel_locked: false,
+                description: 'Incoming webhook - basic formatting',
+                display_name: 'basic-formatting',
+            };
+
+            cy.apiCreateWebhook(newIncomingHook).then((hook) => {
+                incomingWebhook = hook;
+            });
+
+            offTopicLink = `/${team.name}/channels/off-topic`;
+        });
+
+        cy.apiGetMe().then(me => {
+            sysadminUser = me.user;
+        });
+    });
+
+    beforeEach(() => {
+        cy.visit(`/${testTeam.name}/channels/${testChannel.name}`);
+    });
+
+    describe('MM-T626 Incoming webhook is only image and fallback text', () => {
+        const id = 'MM-T626';
+
+        it('first payload', () => {
+            const search = id + '-1';
+            const payload = {text: search, "attachments":[{"fallback": "fallback text", "image_url":"https://cdn.pixabay.com/photo/2017/10/10/22/24/wide-format-2839089_960_720.jpg"}]};
+
+            cy.postIncomingWebhook({url: incomingWebhook.url, data: payload});
+
+            cy.getLastPost().within((el) => {
+                cy.get('.post-message__text').should('have.text', search);
+                cy.get('.attachment__image').should('have.attr', 'src', 'http://localhost:8065/api/v4/image?url=https%3A%2F%2Fcdn.pixabay.com%2Fphoto%2F2017%2F10%2F10%2F22%2F24%2Fwide-format-2839089_960_720.jpg');
```

Comment:
This will fail when running test other than `localhost:8065`. Also, you may use `encodeURIComponent` to deal with URI encoding.

You might need the ff:
```javascript
const baseUrl = Cypress.config('baseUrl');
const imageUrl = 'https://cdn.pixabay.com/photo/2017/10/10/22/24/wide-format-2839089_960_720.jpg';
const imageSrc = `${baseUrl}/api/v4/image?url=${encodeURIComponent(imageUrl)}`;
```